src/results/existing_boiler.jl

```diff
@@ -57,20 +62,20 @@ function add_existing_boiler_results(m::JuMP.AbstractModel, p::REoptInputs, d::D
     else
         BoilerToHotTESKW = zeros(length(p.time_steps))
     end
-	r["year_one_thermal_to_tes_series_mmbtu_per_hour"] = round.(value.(BoilerToHotTESKW / KWH_PER_MMBTU), digits=3)
+	r["thermal_to_tes_series_mmbtu_per_hour"] = round.(value.(BoilerToHotTESKW / KWH_PER_MMBTU), digits=3)
```

Comment:
(another unsolicited approval but pointing out that it's a little different from electric storage). For electric techs to storage, we say "electric_to_storage" while we're using "tes" for thermal (energy) storage. Also, we're currently using "storage_to_" for hot/cold storage production to load, so we're a little inconsistent between "to_tes" and "storage_to". I will leave it up to you to change. I feel like we've added a lot on your task :-).

Reply:
After realizing for production to electric storage (battery), we do "_to_battery", not "to_storage" like I said in the comment above, I'm less strong on changing anything. In theory we could still replace "battery" and "tes" with "storage" because you'll also know the tech type and whether the units are kw (electric), mmbtu_per_hour (hot thermal), or ton (cold thermal).

Reply:
I agree that `to_storage` is more consistent. I made this change in https://github.com/NREL/REopt.jl/pull/160/commits/d823d21ddb945ae29a5b9b44236a002f6fff0b2b

Reply:
Thanks for going the extra mile on that!